dev/changelog.yml

```diff
@@ -0,0 +1,15 @@
+types:
```

Comment:
Should we leave a comment at the top of the file to just indicate it's by `check_pr_title.py`?